src/components/templates/FaqTemplate.js

```diff
@@ -1,10 +1,54 @@
-import React from 'react';
-import { Link } from 'gatsby';
+import React from "react";
+import { Link } from "gatsby";
+import Heading from "../atoms/Heading";
+import Fontain from "../atoms/Fontain";
+import FontainImg from "../../../static/assets/Fontain.png";
+import FaqPair from "../molecules/FaqPair";
+import styled from "@emotion/styled";
+
+const ContentContainer = styled.div({
+  display: "flex",
+  flexDirection: "column",
+  margin: "0 29.17%"
+});
 
 const FaqTemplate = () => (
   <div>
-    <p>FAQ!</p>
     <Link to="/">Go Back</Link>
```

Comment:
pull the latest develop and implement my `NavBar` here.. also delete this `<Link />`